.github/workflows/kanister-image-build.yaml

```diff
@@ -0,0 +1,50 @@
+name: build-kanister-image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Image tag in the format vx.x.x'
+        required: true
+        type: string
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: kanisterio/build
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    # TODO: Enable following when we want to automate this workflow on push to master branch
+    # - uses: actions/checkout@v3
+    #   with:
+    #     fetch-depth: 0
+    # - uses: tj-actions/changed-files@v35
+    #   name: Get changed files
+    #   id: changed-files
+    #   with:
+    #     files: docker/build/Dockerfile
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+      # TODO: Uncomment following when we enable changed-files step
+      # if: steps.changed-files.outputs.any_changed == 'true'
```

Comment:
[Future enhancement] Could we improve this by moving the changed-files out into a separate job? Is it possible to run a job and pass the output into this one or do we have to use the step output everywhere?

Reply:
+1

Reply:
added new job